dependencies.props

```diff
@@ -21,15 +21,15 @@
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <CoreFxExpectedPrerelease>preview2-25319-04</CoreFxExpectedPrerelease>
-    <CoreClrExpectedPrerelease>preview2-25316-03</CoreClrExpectedPrerelease>
+    <CoreFxExpectedPrerelease>preview1-25320-01</CoreFxExpectedPrerelease>
+    <CoreClrExpectedPrerelease>preview1-25319-05</CoreClrExpectedPrerelease>
```

Comment:
`master` is building for 2.1.0 now, so the prerelease spec is reset to preview1. It's clearer in the full package version upgrade: `2.0.0-preview2-25319-02 -> 2.1.0-preview1-25320-02`.